mercury.spec

```diff
@@ -74,7 +64,6 @@ Mercury
 %package devel
 Summary:  Mercury devel package
 Requires: %{name}%{?_isa} = %{version}-%{release}
-Requires: libfabric-devel >= 1.9.0-5
```

Comment:
Please don't remove this.  This ensures that the required `libfabric-devel` package is installed when one installs `mercury-devel`.  Unless there are valid use-cases to install `mercury-devel` and not need/want `libfabric-devel`.

Reply:
Right that's not needed, mercury does not require libfabric headers to build against it. Only DAOS uses libfabric directly to do some special handling in the control plane but DAOS's rpm spec file does specify the libfabric version it wants to use.

Reply:
> mercury does not require libfabric headers to build against

Ah.  I see.  Then indeed, this `BR:` is not necessary.